src/migrations/003-create-table-pages.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+exports.up = (knex) => {
+  return knex.schema.createTableIfNotExists('pages', function (table) {
+    table.text('id');
```

Comment:
This needs to be the primary key for the table.

http://knexjs.org/#Schema-primary

Reply:
See also: http://www.postgresqltutorial.com/postgresql-primary-key/